sandboxed_containers/uninstalling-sandboxed-containers.adoc

```diff
@@ -6,16 +6,26 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+You can uninstall the {sandboxed-containers-first} by using either the {product-title} web console or OpenShift CLI (`oc`). Follow the steps below in the order that they are presented.
```

Comment:
@tshwartz I suggest changing "Follow the steps below in the order that they are presented." to "Both procedures are explained below." (or something similar). The reason I suggest this change is because it seems you are asking them to perform BOTH procedures instead of one OR the other.

Reply:
Fixed.